src/main/cpp/RobotContainer.cpp

```diff
@@ -1,67 +1,315 @@
-// Copyright (c) FIRST and other WPILib contributors.
-// Open Source Software; you can modify and/or share it under the terms of
-// the WPILib BSD license file in the root directory of this project.
+
+
+/****************************************** CREDITS: **********************************************************************************************************************
+
+		This code was written by Jacob Hagberg with a lot of help from Tyler Sieford.
+		Other members of the programming team who helped a lot were Austin and Eddie.
+		This year was a success as far as programming is consirned, as well as the building and electronic aspect.
+		I enjoyed this year a lot working with these guys to built a great robot. We had a great season and I hope for many more to come.
+
+
+		I decided to heavily comment this code in the hopes that it will be a good programming resource for the next robotics season, and maybe even beyond that.
+		I hope that this is helpful and good luck on programming to whoever is reading this.
+**************************************************************************************************************************************************************************
+*/
 
 #include "RobotContainer.h"
+#include "iostream"
+#include "frc2/command/WaitCommand.h"
+#include "frc2/command/SequentialCommandGroup.h"
+#include "frc2/command/ParallelCommandGroup.h"
+
+using namespace std;
+
+/*
+Ok so this is the robot container. This is where we take all the functions and code from the subsystems and make them respond to control
+inputs as well as to get them to work together. We mainly do this through commands
+
+*/
```

Comment:
Some of this teaching style comments would be best to put into either a tutorial in the [Documentation repo](https://github.com/FRCTeam2501/Documentation) or a presentation to teach the next students.